addons/token-exchange/agent_mirrorpeer_controller.go

```diff
@@ -59,8 +71,13 @@ const (
 	RBDVolumeReplicationClassNameTemplate = "rbd-volumereplicationclass-%v"
 	RBDReplicationSecretName              = "rook-csi-rbd-provisioner"
 	DefaultMirroringMode                  = "snapshot"
+	RamenLabelTemplate                    = "ramendr.openshift.io/%s"
+	StorageIDKey                          = "storageid"
+	ReplicationIDKey                      = "replicationid"
 )
 
+var clusterFSIDs = make(map[string]string)
```

Comment:
don't use global variable. define it in function scope and pass it to other places as args.

Reply:
done